### PR TITLE
Fix #1850: Made execution time show only if greater than 0

### LINKED ISSUE #1850: Display execution time only when greater than zero.
Currently, all the types of execution time is shown on the frontend under `my submissions`. We would like to show only the execution time which is greater than zero.

Helpful link: https://stackoverflow.com/questions/18403835/how-to-make-less-than-or-greater-than-comparison-in-angularjs?answertab=active#tab-top

### PR DESCRIPTION
**Pull request by dzpzee**
In "my submissions" page

#1850

### CODE REVIEW

File: frontend/src/views/web/challenge/my-submission.html [hunk 1]
```diff
@@ -92,7 +92,7 @@
                                             <td ng-if="key.publication_url == ''"> None </td>
 
                                             <td class="val-style capitalize" ng-class="key.status">{{key.status}}</td>
-                                            <td class="align-center">{{key.execution_time}}</td>
+                                            <td class="align-center" ng-show="key.execution_time > 0">{{key.execution_time}}</td>
```
Comment (RishabhJain2018) [inline frontend/src/views/web/challenge/my-submission.html]: Please add a default value `None` which will be displayed when the execution time is < 0.

Comment (dzpzee) [reply frontend/src/views/web/challenge/my-submission.html]: Done.

Comment (dzpzee) [reply frontend/src/views/web/challenge/my-submission.html]: Done.

**Comment by dzpzee** [comment]
Ok. I think that worked.

**Comment by RishabhJain2018** [comment]
LGTM :) Thanks for the fix!

**Pull request merged**